modules/avatar/react/lib/Avatar.tsx

```diff
@@ -55,9 +60,8 @@ type AvatarOverload = {
 
 const StyledContainer = styled('button', {
   shouldForwardProp: prop => isPropValid(prop) && prop !== 'size',
-})<Pick<AvatarProps, 'size' | 'onClick'>>(
+})<Pick<AvatarProps, 'size' | 'onClick' | 'imageLoaded' | 'url'>>(
```

Comment:
Remove `imageLoaded` from the `AvatarProps` interface and make this change instead:

```suggestion
})<Pick<AvatarProps, 'size' | 'onClick' | 'url'> & { imageLoaded: boolean }>(
```